projects/plugins/jetpack/extensions/shared/create-block-from-inner-blocks-template.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlocksFromInnerBlocksTemplate, createBlock } from '@wordpress/blocks';
+
+/**
+ * This is a helper function that acts as a fallback
+ * for the core `createBlocksFromInnerBlocksTemplate()` function.
+ * This function is not available for the different Jetpack versions
+ * that a WordPress site currently could get.
+ *
+ * @param {Array} innerBlocksOrTemplate - Nested blocks or InnerBlocks templates.
+ *
+ * @returns {Object[]} Array of Block objects.
+ */
+export default function createBlocksFromTemplate(
```

Comment:
Should we keep the naming consistent, so that we know this is a fallback for `createBlocksFromInnerBlocksTemplate`?

Reply:
Actually, I changed the name on purpose because I want to make it evident that it's a different function than the core one. Said that I'm not sure if it's good or not.

Reply:
How about `jetpackCreateBlocksFromInnerBlocksTemplate` then?